README.md

```diff
@@ -48,6 +48,34 @@ More information by:
 
 If you want to change colors in vim as you wish, you can see the list of highlight groups [here](https://github.com/objectionary/eo-vim/blob/master/syntax/README.md).
 
+## Highlight setting
```

Comment:
@ivan-egorov42 we already have "Configuration" title. "Highlight setting" doesn't look like a good fit for the same level heading